lib/src/provider/asset_picker_provider.dart

```diff
@@ -56,13 +56,25 @@ abstract class AssetPickerProvider<Asset, Path> extends ChangeNotifier {
   /// 销毁时重置所有内容
   @override
   void dispose() {
+    _mounted = false;
     _isAssetsEmpty = false;
     _paths.clear();
     _currentPath = null;
     _currentAssets.clear();
     super.dispose();
   }
 
+  @override
+  void notifyListeners() {
+    if (_mounted) {
+      super.notifyListeners();
+    }
+  }
```

Comment:
This part is actually optional since it is not needed to fix #492 issue.
However `notifyListeners()` will throw an exception if the provider was disposed, so i thought it was still good to put a flag on it.